tron/core/actionrun.py

```diff
@@ -200,16 +200,22 @@ def to_json(state_data: dict) -> Optional[str]:
     def from_json(state_data: str):
         """Deserialize the ActionRunAttempt instance from a JSON string."""
         try:
-            json_data = json.loads(state_data)  # here we will need json.loads because "attempts" is a string
-            for k in json_data:
-                if k == "start_time" or k == "end_time":
-                    json_data[k] = datetime.datetime.fromisoformat(json_data[k]) if json_data[k] else None
-                elif k == "command_config":
-                    json_data[k] = ActionCommandConfig.from_json(json_data[k])
+            json_data = json.loads(state_data)
+            deserialized_data = {
+                "command_config": ActionCommandConfig.from_json(json_data["command_config"]),
+                "start_time": datetime.datetime.fromisoformat(json_data["start_time"])
+                if json_data["start_time"]
+                else None,
```

Comment:
will `black` still mangle this if we wrap this in parens like
```
"start_time": (
    datetime.datetime.fromisoformat(json_data["start_time"])
    if json_data["start_time"]
    else None
),
```

(if it does, then oh well)

Reply:
(but if it doesn't, it'd be nice to apply this to the other places where `black` output weird formatting like this)

Reply:
it will be like this 
```
                "start_time": (
                    datetime.datetime.fromisoformat(json_data["start_time"]) if json_data["start_time"] else None
                ),
```